packages/frontend/core/src/modules/integration/entities/calendar.ts

```
@@ -45,19 +71,87 @@ export class CalendarIntegration extends Entity {
+    return this.subscriptions$.map(subscriptions =>
+      subscriptions.find(sub => sub.url === url)
+    );
+  }
+  contents$ = LiveData.computed(get => {
+    const subscriptions = get(this.subscriptions$);
+    return subscriptions.map(sub => ({
+      url: sub.url,
+      content: get(sub.content$),
+    }));
+  });
+  eventsByDateMap$ = LiveData.computed(get => {
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)